about.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="index.css">
+       <title>All about the PSS conference</title>
+</head>
+<body>
+    <section>
+        <div class="link-bar">
+            <ul>
+                <li><a href=""><img src="./images/Vector4.png" class="alt-img"></a></li>
+                <li><a href=""><img src="./images/Vector2.png" class="alt-img"></a></li>
+                <li>Ikoote</li>
+                <li>MyPage</li>
+                <li>Logout</li>
+            </ul>
+        </div>
+    </section>
+    <section class="menu-a">
+        <div class="nav-menu-a">
+            <div><a href="./index.html"><img src="./images/img-logo.jpg">ikoote PSS</a></div>
+            <div class="nav-a">
+                <a href="./index.html">Home</a>
+                <a href="program">Program</a>
+                <a href="#">Join</a>
+                <a href="#sponsors">Sponsors</a>
+                <a href="#speakers">Feature Speakers</a>
+                <a href=""><button type="button" class="btn">PSS campaign</button></a>
+            </div>
```

Comment:
[OPTIONAL] This looks good 💯 but would look even better if the tags were centered horizontally inside of the `div` (on the same line) 📏 . Take a look 👇🏽 

![Screenshot 2023-02-09 at 07 33 52](https://user-images.githubusercontent.com/45739963/217813991-beba6c77-9f0b-4c50-8416-04493f172c34.png)